Fails to build when using parallel make

I cannot compile `playerctl` using parallel `make`:

```
alexander@localhost:.../playerctl$ make -j 4
make  all-recursive
make[1]: Entering directory '/home/alexander/src-ext/playerctl'
Making all in playerctl
make[2]: Entering directory '/home/alexander/src-ext/playerctl/playerctl'
/usr/bin/gdbus-codegen --generate-c-code playerctl-generated mpris-dbus-interface.xml
make[2]: *** No rule to make target 'playerctl-generated.h', needed by 'all'.  Schluss.
make[2]: *** Warte auf noch nicht beendete Prozesse...
make[2]: Leaving directory '/home/alexander/src-ext/playerctl/playerctl'
Makefile:427: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/alexander/src-ext/playerctl'
Makefile:358: recipe for target 'all' failed
make: *** [all] Error 2
```

Building using the non-parallel `make` succeeds:

```
alexander@localhost:../playerctl$ make
make  all-recursive
make[1]: Entering directory '/home/alexander/src-ext/playerctl'
Making all in playerctl
make[2]: Entering directory '/home/alexander/src-ext/playerctl/playerctl'
make  all-am
make[3]: Entering directory '/home/alexander/src-ext/playerctl/playerctl'
  CC       playerctl-player.lo
  CC       playerctl-generated.lo
  CCLD     libplayerctl-1.0.la
  CC       playerctl-playerctl-cli.o
  CCLD     playerctl
  GISCAN   Playerctl_1_0_gir
  GISCAN   Playerctl-1.0.gir
  GEN      playerctl-1.0.pc
  GICOMP   Playerctl-1.0.gir
make[3]: Leaving directory '/home/alexander/src-ext/playerctl/playerctl'
make[2]: Leaving directory '/home/alexander/src-ext/playerctl/playerctl'
Making all in doc
make[2]: Entering directory '/home/alexander/src-ext/playerctl/doc'
Making all in reference
make[3]: Entering directory '/home/alexander/src-ext/playerctl/doc/reference'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/alexander/src-ext/playerctl/doc/reference'
make[3]: Entering directory '/home/alexander/src-ext/playerctl/doc'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/home/alexander/src-ext/playerctl/doc'
make[2]: Leaving directory '/home/alexander/src-ext/playerctl/doc'
make[2]: Entering directory '/home/alexander/src-ext/playerctl'
make[2]: Leaving directory '/home/alexander/src-ext/playerctl'
make[1]: Leaving directory '/home/alexander/src-ext/playerctl'
```

--- a/playerctl/Makefile.am
+++ b/playerctl/Makefile.am
@@ -29,7 +29,7 @@ source_c = \
 
 DISTCLEANFILES += playerctl-version.h
 
-playerctl-generated.c:
+playerctl-generated.h playerctl-generated.c:
 	$(GDBUS_CODEGEN) --generate-c-code playerctl-generated mpris-dbus-interface.xml
 
 BUILT_SOURCES += \
@@ -74,6 +74,7 @@ INTROSPECTION_COMPILER_ARGS =
 
 if HAVE_INTROSPECTION
 
+Playerctl_1_0_gir: libplayerctl-1.0.la
 Playerctl-1.0.gir: libplayerctl-1.0.la
 Playerctl_1_0_gir_INCLUDES = GObject-2.0
 Playerctl_1_0_gir_CFLAGS = $(AM_CPPFLAGS)

